pyproject.toml

```diff
@@ -68,13 +68,12 @@ docs = [
 [project.urls]
 Repo = "https://github.com/radical-ai/torch-sim"
 
-[tool.setuptools.packages.find]
-include = ["torch_sim*"]
-exclude = ["examples*", "tests*"]
-
 [build-system]
-requires = ["setuptools==77.0.3"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling>=1.8.0"]
```

Comment:
latest `hatchling` as of now is [1.27.0](https://pypi.org/project/hatchling/1.27.0) so let's pin to that or newer